app/processing/apply-dual-accounting.js

```diff
@@ -0,0 +1,28 @@
+const { FDMR, BPS } = require('../constants/schemes')
+const { DOM00, DOM01, DOM10 } = require('../constants/domestic-fund-codes')
+
+const addDualAccounting = (paymentRequests, previousPaymentRequests) => {
+  for (const paymentRequest of paymentRequests) {
+    for (const invoiceLine of paymentRequest.invoiceLines) {
+      if (paymentRequest.schemeId === FDMR || paymentRequest.schemeId === BPS) {
```

Comment:
Will that work if there are multiple payment requests coming through?

Reply:
The payment requests will be the result of the post payment adjustment calculation.  So will either just be one if first payment or a post payment adjustment that doesn't need splitting.  If multiple just means it had to be split across multiple ledgers.  But because it's the same payment request essentially, the scheme will always be the same